tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -407,18 +407,19 @@ module('Registries | Acceptance | overview.overview', hooks => {
         await visit(`/${reg.id}/`);
 
         assert.dom('[data-test-edit-button="license"]').isVisible();
+        // await pauseTest();
         await click('[data-test-edit-button="license"]');
 
         assert.dom('[data-test-license-edit-form]').isVisible();
-        await selectSearch('[data-test-select-license]', 'MIT');
+        await selectSearch('[data-test-power-select-dropdown]', 'MIT');
         assert.dom('.ember-power-select-options').hasText('MIT License');
-        await selectSearch('[data-test-select-license]', 'No');
+        await selectSearch('[data-test-power-select-dropdown]', 'No');
         assert.dom('.ember-power-select-options').hasText('No license');
-        await selectChoose('[data-test-select-license]', 'No license');
+        await selectChoose('[data-test-power-select-dropdown]', 'No license');
 
         await click('[data-test-save-license]');
 
-        const missingFields = t('app_components.license_picker.fields.copyrightHolders').toString();
+        const missingFields = 'copyrightHolders';
```

Comment:
Currently we can't inject `intl` service into validation functions, therefore there is no clean way of getting the translation for the fields. We should revisit that when ember-changeset-validation supports this. See [this issue on ember-changeset-validations](https://github.com/poteto/ember-changeset-validations/issues/197).